alg/crc32c.c

```diff
@@ -63,7 +63,7 @@ init(void)
 
 	/* Fill in tables. */
 	for (i = 0; i < 256; i++) {
-		r = reverse(i);
+		r = reverse(i & 0xffff);
```

Comment:
Not sure why you're masking to take the bottom 16 bits here... in any case, I'd go with a cast to `(uint32_t)`.